models/v1beta1/connection/connection_helper_test.go

```
@@ -0,0 +1,126 @@
+package connection
```

golangci-lint (2.12.2)

[error] ST1000: at least one file in a package should have a package comment

(staticcheck)